packages/react-core/src/next/components/Select/examples/SelectTypeahead.tsx

```diff
@@ -0,0 +1,191 @@
+import React from 'react';
+import { Select, SelectOption, SelectList, SelectOptionProps } from '@patternfly/react-core/next';
+import {
+  MenuToggle,
+  MenuToggleElement,
+  TextInputGroup,
+  TextInputGroupMain,
+  TextInputGroupUtilities,
+  Button
+} from '@patternfly/react-core';
+import TimesIcon from '@patternfly/react-icons/dist/esm/icons/times-icon';
+
+const initialSelectOptions: SelectOptionProps[] = [
+  { itemId: 'Alabama', children: 'Alabama' },
+  { itemId: 'Florida', children: 'Florida' },
+  { itemId: 'New Jersey', children: 'New Jersey' },
+  { itemId: 'New Mexico', children: 'New Mexico' },
+  { itemId: 'New York', children: 'New York' },
+  { itemId: 'North Carolina', children: 'North Carolina' }
+];
```

Comment:
Hmm... for some reason, I'm not seeing these items in the list (or the `placeholder`) for this or the multi typeahead demo. Maybe that's an issue with surge on my end?

Reply:
That's odd I am also not seeing these on the preview (the code is a commit behind when I looked). Maybe with the next commit it will update the surge properly.